mxcubeweb/routes/login.py

```diff
@@ -80,24 +68,22 @@ def login_info():
         {'synchrotron_name': synchrotron_name,
         'beamline_name': beamline_name,
         'loginType': loginType,
-        'loginRes': {'status':{ 'code': 'ok', 'msg': msg },
+        'loggedIn': True,
         'Proposal': proposal, 'session': todays_session,
         'local_contact': local_contact,
         'person': someone,
         'laboratory': a_laboratory']}}
 
         Status code set to:
         200: On success
-        409: Error, could not log in
+        200: Error, could not log in, {"loggedIn": False}
         """
-        user, res = app.usermanager.login_info()
 
-        # Redirect the user to login page if for some reason logged out
-        # i.e. server restart
-        if not user:
-            response = redirect("/login", code=302)
-        else:
+        try:
+            res = app.usermanager.login_info()
             response = jsonify(res)
+        except Exception:
```

Comment:
What about having a more specific Exception type? See `mxcubeweb/core/components/user/usermanager.py` line 262 where the exception is raised.

```suggestion
        except NotAuthenticated:
```